README.md

```diff
@@ -125,21 +197,56 @@ to re-open the REPL to change it.
 
 ## Commands
 
-You can execute it by entering the following command at the command palette
-<kbd>Ctrl</kbd>+<kbd>P</kbd> (<kbd>Cmd</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> on
-MacOS).
+An easy way to see what commands are offered by the extension in the currently
+open file, you can invoke VSCode Command Palette and search for commands with
+prefix `OCaml:`:
+
+![commands](./readme-assets/commands.png)
```

Comment:
Broken path. Should be `./assets`?

Reply:
Same thing with the `![pick sandbox]` image.

Reply:
strange, the markdown seems rendered fine here: https://github.com/ulugbekna/vscode-ocaml-platform/tree/update-readme